src/tests/Common/Directory.Build.targets

```diff
@@ -171,34 +171,31 @@
       <RunTimeDependencyCopyLocal Include="$(CoreDisToolsLibrary)" Condition="$(CopyCoreDisToolsToCoreRoot)" />
     </ItemGroup>
 
-<<<<<<< HEAD
+
     <PropertyGroup>
-      <IlcDir>$(CoreCLRArtifactsPath)ilc</IlcDir>
+      <!-- TODO-LLVM: used to get the windows.x64.configuration build folder for tests, obviously there's a right way to do this -->
+      <HostCoreCLRArtifactsPath>$([MSBuild]::NormalizeDirectory('$(RepoRoot)', 'artifacts', 'bin', 'coreclr', '$(HostOS).$(BuildArchitecture).$(CoreCLRConfiguration)'))</HostCoreCLRArtifactsPath>
+      <IlcDir>$(HostCoreCLRArtifactsPath)ilc</IlcDir>
     </PropertyGroup>
 
     <!-- When cross-compiling tests (such as is the case for Wasm), we depends on the host's ILC being built. -->
     <!-- This is a workaround for ILC's project not supporting cross-targeting. Remove once that's implemented. -->
     <PropertyGroup Condition="'$(TargetArchitecture)' != '$(BuildArchitecture)'">
-      <IlcDir>$([MSBuild]::NormalizeDirectory('$(RepoRoot)', 'artifacts', 'bin', 'coreclr', '$(BuildOS).$(BuildArchitecture).$(CoreCLRConfiguration)'))ilc</IlcDir>
+      <IlcDir>$([MSBuild]::NormalizeDirectory('$(RepoRoot)', 'artifacts', 'bin', 'coreclr', '$(HostOS).$(BuildArchitecture).$(CoreCLRConfiguration)'))ilc</IlcDir>
     </PropertyGroup>
 
     <Error Condition="!Exists('$(IlcDir)')"
-           Text="'$(IlcDir)' does not exist. Subsets clr.jit, clr.wasmjit and nativeaot.ilc for $(BuildOS) $(BuildArchitecture) in $(CoreCLRConfiguration) configuration must be built before building this project." />
+           Text="'$(IlcDir)' does not exist. Subsets clr.jit, clr.wasmjit and nativeaot.ilc for $(HostOS) $(BuildArchitecture) in $(CoreCLRConfiguration) configuration must be built before building this project." />
```

Comment:
GH doesn't let me make a suggestion here because of deletions, but this should be simplified:
```
    <!-- When cross-compiling tests (such as is the case for Wasm), we depends on the host's ILC being built. -->
    <!-- This is a workaround for ILC's project not supporting cross-targeting. Remove once that's implemented. -->
    <PropertyGroup>
      <HostCoreCLRArtifactsPath>$([MSBuild]::NormalizeDirectory('$(RepoRoot)', 'artifacts', 'bin', 'coreclr', '$(HostOS).$(BuildArchitecture).$(CoreCLRConfiguration)'))</HostCoreCLRArtifactsPath>
    </PropertyGroup>

    <Error Condition="!Exists('$(HostCoreCLRArtifactsPath)')"
           Text="'$(HostCoreCLRArtifactsPath)' does not exist. Subsets clr.jit, clr.wasmjit and clr.tools for $(HostOS) $(BuildArchitecture) in $(CoreCLRConfiguration) configuration must be built before building this project." />
```
(Also includes a fix for the error message)